src/main/java/net/datafaker/providers/base/Number.java

```diff
@@ -59,6 +59,16 @@ public int numberBetween(final int min, final int max) {
         return (int) numberBetween(realMin, (long) realMax);
     }
 
+    /**
+     * @param min the lower bound (include min)
+     * @param max the upper bound (not include max)
+     * @return a random number on faker.number() between min and max
+     * if min = max, return min
+     */
+    public double numberBetween(double min, double max) {
+        return faker.random().nextDouble(min, max);
```

Comment:
it looks like nothing prevents from calling it like
```java
faker.number().numberBetween(123.456, 1.0);
```
what is expected output of it?
based on name the result should be somewhere between the boudaries

Reply:
The same as what the other methods return.

```
faker.number().numberBetween(10, 0) 
```

Also returns a number between 0 and 10.

Reply:
would be great to have a test for this case as well

Reply:
Fair enough, I added it.